components/brave_search/renderer/brave_search_renderer_utils.cc

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/components/brave_search/renderer/brave_search_renderer_utils.h"
+
+#include <string>
+#include <vector>
+
+#include "base/no_destructor.h"
+#include "url/gurl.h"
+
+namespace {
+
+static base::NoDestructor<std::vector<std::string>> g_vetted_hosts(
```

Comment:
just use `base::MakeFixedFlatSet` and `base::Contains` instead of linear search (see https://groups.google.com/a/chromium.org/g/chromium-dev/c/lVsznaNa_Bk)

Reply:
ok thanks, I just copied this from the existing API to be used by both APIs, but happy to make the improvement

Reply:
fixed, thanks again